src/network/udpclient.cpp

```diff
@@ -174,6 +174,7 @@ void Network::sendAccel(float* vector, uint8_t sensorId) {
     if(DataTransfer::beginPacket()) {
         DataTransfer::sendPacketType(PACKET_ACCEL);
         DataTransfer::sendPacketNumber();
+        DataTransfer::sendByte(sensorId);
```

Comment:
You'll break all servers, please add this as the last value in the message

Reply:
> You'll break all servers, please add this as the last value in the message
Do you mind elaborating a little? It worked either way for me and the way it previously was seemed to be the convention. I changed it regardless just wondering.

Reply:
> You'll break all servers, please add this as the last value in the message

Sensor ID comes after packet number. Why it should be last?